src/components/Forms/Configs/AWSConfigsFormEditor.tsx

```diff
@@ -25,6 +26,46 @@ export default function AWSConfigsFormEditor({
 
   return (
     <>
+      {/* this a top level schema field, not nested under http */}
+      <FormikAutocompleteDropdown
```

Comment:
Can we extract this into a standalone component ?

Reply:
Done